ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/ClaimDocumentMapperTest.java

```diff
@@ -71,9 +72,13 @@ public void mapFrom() {
         ClaimDocument claimDocument = claimDocumentMapper.from(ccdCollectionElement);
         CCDClaimDocument ccdClaimDocument = ccdCollectionElement.getValue();
 
-        assertEquals(ccdClaimDocument.getDocumentName(), claimDocument.getDocumentName());
         assertEquals(URI.create(ccdClaimDocument.getDocumentLink().getDocumentUrl()),
             claimDocument.getDocumentManagementUrl());
+
+        assertEquals(URI.create(ccdClaimDocument.getDocumentLink().getDocumentUrl()),
+            claimDocument.getDocumentManagementUrl());
```

Comment:
Duplicates the previous assertion. I guess this is meant to be about the binary URL